### DISCUSSION TITLE
公告怎么带图片

### DISCUSSION THREAD

**Discussion by fangxin810425** [Mrs4s/go-cqhttp#1405] [Q&A]
在使用_send_group_notice添加群公告时怎么才可以带上图片？

**Comment by qianjunakasumi** [comment]
该功能已经实现，文档并未更新。请您使用

image 字段，使用资源定位符

例：

GET localhost:11451/_send_group_notice?group_id=1145141919&content=我是公告&image=/mnt/114514.png

https://github.com/Mrs4s/go-cqhttp/blob/8d26e3aec4ce66d41ed422ef37ca407f6c92ba8d/coolq/api.go#L1003

**Reply by wdvxdr1123** [reply]
cc @ishkong
